### PR TITLE
Update multiple URL rewrites

### PR DESCRIPTION
**Pull request by hickskenh**
per gh #1, added multiple rewrites topic and image, updated marketing toc, and added a cross reference in the create rewrites topic.

## Purpose of this pull request

This pull request (PR) is to fix GH1, update multiple URL rewrites. Added a new topic and image, updated the marketing toc, and added a cross reference in the create rewrite topic.

## Magento release version

- [ ] 2.4.x

   Specify a patch release number, if applicable:

- [X] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/marketing/url-rewrite.html
- https://docs.magento.com/user-guide/marketing/url-rewrite-update-all.html   (new)

## Additional information

Fixes GH issue 1. staging build 531.

whatsnew
Added a new topic for updating [Multiple URL Rewrites](https://docs.magento.com/user-guide/marketing/url-rewrite-update-all.html).

### CODE REVIEW

File: src/marketing/url-rewrite-update-all.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Multiple URL Rewrites
+---
+
+To create a single URL rewrite, refer to [Creating URL Rewrites]({% link marketing/url-rewrite-create.md %}).
+
+To quickly update URL rewrites for multiple or all products simultaneously, follow the steps below.
```
Comment (leslietilling) [inline src/marketing/url-rewrite-update-all.md:7]:
```suggestion
You can quickly update URL rewrites for multiple or all products simultaneously using the following steps.
```

File: src/marketing/url-rewrite-update-all.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Update multiple URL rewrites
```
Comment (leslietilling) [inline src/marketing/url-rewrite-update-all.md:9]: You need a blank line here to pass the MD lint check.

File: src/marketing/url-rewrite-update-all.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+1. On the _Admin_ sidebar, go to **Catalog** > **Products**.
+1. Select all of the products for which you want to update URL rewrites.
+1. Under _Actions_, choose **Update attributes** to update multiple or all rewrites.
+1. Under _PRODUCTS INFORMATION_, choose the **Websites** tab.
```
Comment (leslietilling) [inline src/marketing/url-rewrite-update-all.md:13]:
```suggestion
1. Under _PRODUCTS INFORMATION_, click the **Websites** tab.
```

**Comment by hickskenh** [comment]
running tests

**Comment by dshevtsov** [comment]
Changed the URLs of affected pages and whatsnew to point the 2.3 site.

**Pull request merged**